src/Serilog.Sinks.OpenTelemetry/Sinks/OpenTelemetry/ProtocolHelpers/PrimitiveConversions.cs

```diff
@@ -128,11 +128,9 @@ public static AnyValue ToOpenTelemetryMap(StructureValue value)
         
         foreach (var prop in value.Properties)
         {
-            if (seen.Contains(prop.Name))
+            if (!seen.Add(prop.Name))
```

Comment:
the `!` and the continue makes stuff a bit cryptic
having a block guarded by the add succeeding whhoudl be better?
(and the body of that should all be inlined as a one liner in my book)

Reply:
Does look better 👍 thanks!

Reply:
not drring it pushed this side yet?